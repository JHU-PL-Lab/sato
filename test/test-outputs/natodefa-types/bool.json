[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "a" ],
          "err_match_val": "0",
          "err_match_expected": "Boolean",
          "err_match_actual": "Integer"
        }
      ],
      [
        "Error_match",
        {
          "err_match_aliases": [ "b" ],
          "err_match_val": "input",
          "err_match_expected": "Boolean",
          "err_match_actual": "Integer"
        }
      ]
    ],
    "err_input_seq": [ 0 ],
    "err_location": "a and b",
    "err_steps": 112
  }
]