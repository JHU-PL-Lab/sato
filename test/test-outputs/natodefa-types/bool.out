Type errors for:
- Input sequence : [0]
- Found at expr  : a and b
--------------------
* Value    : a = 0
* Expected : Boolean
* Actual   : Integer
--------------------
* Value    : b = input
* Expected : Boolean
* Actual   : Integer
2 errors found. No further control flows exist.
